Source/setmaps.cpp

```diff
@@ -190,6 +191,33 @@ void LoadSetMap()
 		DRLG_SetMapTrans("Levels\\L1Data\\Vile1.DUN");
 		InitNoTriggers();
 		break;
+	case SL_TEST:
+		char dunbuf[100];
+		char palbuf[100];
+		sprintf(palbuf, "derp%d.pal", paletteNum);
+		sprintf(dunbuf, "derp%d.dun", tilesetNum);
+		if (tilesetNum == 0) {
+			LoadPreL1Dungeon(dunbuf, 0, 0);

```

Comment:
It doesn't really make sens to load the same map as the pre dungion layout, the pre-dungion is a .dun where only the tiles are read, and where the level chan switch between the tow layout by way of leavers in the map (see level 16)